x-pack/solutions/search/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines.tsx

```diff
@@ -65,6 +65,28 @@ export const SearchIndexPipelines: React.FC = () => {
   const { makeRequest: revertPipeline } = useActions(RevertConnectorPipelineApilogic);
   const apiIndex = isApiIndex(index);
   const extractionDisabled = getContentExtractionDisabled(index);
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
+  const pipelinesButton = Array.from(document.querySelectorAll('button')).find(
+    (btn) => btn.textContent?.trim() === 'Pipelines'
+  );
+
+  const onCloseDeleteModal = useCallback(() => {
+    closeDeleteModal();
+    setTimeout(() => {
+      if (buttonRef.current) {
+        buttonRef.current.focus();
+      }
+    }, 0);
```

Comment:
you're using two different intervals for the timeouts. I would pick one and store it in a constant so that they are uniform.